src/_sass/components/videoplayer/_section-controls.scss

```diff
@@ -0,0 +1,28 @@
+.section-controls-container {
+  background-color: $green;
+  padding: 0.25rem;
+  height: 2.5rem;
+  @include flex-container($justify:center);
```

Comment:
I kind of prefer `$justify:space-between`, with the buttons to the far left and right of the container, but I guess it comes down to aesthetics. Not sure which one fits the overall look and feel of the site better.

Reply:
We'll go with `space-between` for now and can always alter. We don't have anything else quite the same functionally or aesthetically, so I think either is a good option.